tests/www/test_views.py

```diff
@@ -2200,6 +2200,7 @@ def test_action_logging_post(self):
             only_failed="false",
         )
         resp = self.client.post("clear", data=form)
+        # sometimes fail due to shared db?
```

Comment:
Can you remove this commend? Flaky tests are a problem and we try to limit it in many ways.